src/vs/workbench/contrib/codeActions/browser/codeActionsContribution.ts

```diff
@@ -64,7 +64,7 @@ const codeActionsOnSaveSchema: IConfigurationPropertySchema = {
 			items: { type: 'string' }
 		}
 	],
-	markdownDescription: nls.localize('editor.codeActionsOnSave', 'Run Code Actions for the editor on save. Code Actions must be specified and the editor must not be shutting down. Example: `"source.organizeImports": "explicit" `'),
+	markdownDescription: nls.localize('editor.codeActionsOnSave', 'Run Code Actions for the editor on save. Code Actions must be specified and the editor must not be shutting down. When [auto save](https://code.visualstudio.com/docs/editor/codebasics#_save-auto-save) is set to `afterDelay`, Code Actions will run when the file is saved explicitly, not when saved automatically. Example: `"source.organizeImports": "explicit" `'),
```

Comment:
for this and all cases below, instead of pointing towards documentation, please point it towards the auto save setting

Reply:
nit: change to `....will be run only when the file is saved explicitly.`

Reply:
Addressed in 2de9ac0 and 242ad96